configuration/price.go

```
@@ -0,0 +1,54 @@
+package configuration
+
+type PricesInfo struct {
+	WeekDayHourlyPrice    float32
+	HoursWeekDay          int
+	WeekendDayHourlyPrice float32
+	HoursWeekendDay       int
+	BhDayHourlyPrice      float32
+	HoursBhDay            int
+}
+
+func (c *Configuration) GetPricesInfo() (*PricesInfo, error) {
+	weekDayPrice, err := c.FindPriceByDay("weekday")
+	if err != nil {
+		return nil, err
+	}
+	excludedWeekDayHoursAmount := 0
+	excludedHours, _ := c.FindRotationExcludedHoursByDay("weekday")
```

why err is discarded here?

---

I just moved this function from cmd package to the config package because it makes more sense to live there, but I didn't change any logic of it as you can see it [here](https://github.com/form3tech-oss/go-pagerduty-oncall-report/blob/master/cmd/generate_report.go#L439)

---

I see, but it is awkward to see err being discarded. I would personally add err handling here.

---

I removed from the function because this error returning from `FindRotationExcludedHoursByDay` was not used anywhere.